src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -165,6 +165,9 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_20230702T105416Z.xml
       relativeToChangelogFile: true
+  - include:
```

Comment:
why adding it before changelog_20230728T141452Z, ho I see... hum maybe you should regenerate it ? hum not really useful by the way